contracts/axelar-gateway/src/contract.rs

```diff
@@ -93,13 +94,17 @@ impl AxelarGatewayInterface for AxelarGateway {
             )
     }
 
-    fn is_message_executed(env: Env, source_chain: String, message_id: String) -> bool {
+    /// Return true if a contract call with the given payload BytesN<32> and source caller info has been executed.
+    pub fn is_message_executed(env: Env, source_chain: String, message_id: String) -> bool {
         let message_approval = Self::message_approval(&env, source_chain, message_id);
 
         message_approval == MessageApprovalValue::Executed
     }
 
-    fn validate_message(
+    /// Validate if a contract call with the given payload BytesN<32> and source caller info is approved,
```

Comment:
How about the other parameters (i.e., `message_id`)? The comment seems to suggest that the `validate_message` function only validates the message with the payload and source caller info.

Reply:
adjusted the docs to better match the evm reference, let me know if it's clearer now